homeassistant/components/synology_dsm/__init__.py

```diff
@@ -122,6 +123,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
     entry.async_on_unload(entry.add_update_listener(_async_update_listener))
 
+    # Setup media source
+    media_source.async_setup(hass)
```

Comment:
we should remove this here, because it would be called for each config entry - just do the webview register within the `async_get_media_source` in `media_source.py`